lib/alexa.js

```diff
@@ -145,13 +145,23 @@ function ValidateRequest() {
         // Validate that this request originated from authorized source.
         if (handlerAppId && (requestAppId !== handlerAppId)) {
             console.log(`The applicationIds don\'t match: ${requestAppId} and ${handlerAppId}`);
-            return callback(new Error('Invalid ApplicationId: ' + handlerAppId));
+            var error = new Error('Invalid ApplicationId: ' + handlerAppId)
+            if(typeof callback === undefined) {
```

Comment:
[typeof](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof) returns a string so this statement will be false

try typeof callback === 'undefined'